samples/mixed/consumer/src/main.rs

```diff
@@ -117,9 +115,11 @@ async fn send_subscribe_request(
     provider_uri: &str,
     entity_id: &str,
     consumer_uri: &str,
-) -> Result<(), Box<dyn std::error::Error>> {
+) -> Result<(), Status> {
```

Comment:
we might not want to use `Status` as the error type here, this makes it tightly bound to tonic. This also doesn't derive from `std::error:Error`, which is usually considered best practice. probably something for another PR though since this is used as an error type all over the place

Reply:
This is in sample code.  Not worth addressing.